src/translations/en.json

```diff
@@ -1,10 +1,21 @@
 {
   "app": {
-    "name": "Talk Partner Sample App",
-    "short_description": "A reference implementation to help app developers to use Talk Partner Edition APIs",
-    "long_description": "A reference implementation to help app developers to use Talk Partner Edition APIs",
-    "installation_instructions": "Thanks for installing this app",
-    "title": "Talk Partner Sample App"
-  },
-  "default": {}
+    "name": {
```

Comment:
This is just reformatting to include the title and values as is the `zendesk` way of doing stuff.

I generated this using `zat translate to_json`